pkg/crc/constants/constants.go

```diff
@@ -221,3 +221,18 @@ func GetDefaultMemory(preset crcpreset.Preset) int {
 		return 9216
 	}
 }
+
+func GetDefaultBundleImageRegistryURL(preset crcpreset.Preset) string {
+	return fmt.Sprintf("//%s/%s:%s", RegistryURI, getImageName(preset), version.GetBundleVersion(preset))
```

Comment:
`docker://%s/%s:%s` for consistency with the URL parsing code there is in the `bundle` package?

Reply:
I'm not even sure `//%s/%s` is a URL, which would be contrary to what the function name implies (`GetDefault...URL`)